### DISCUSSION TITLE
regarding the compatibility of lighthouse deck

### DISCUSSION THREAD

**Discussion by prathyushaavs** [bitcraze/discussions#1430] [Q&A]
how to connect the lighthouse deck to the drones that are not manufactured by bitcraze? how to fly the drones indoor with the lighthouse positioning system?

**Comment by tobbeanton** [comment]
> how to connect the lighthouse deck to the drones that are not manufactured by bitcraze?

Currently we do not provide any way to do this. The code is open though so it would be possible to develop but will require a lot of work.
